src/core/feed-core.js

```diff
@@ -147,13 +135,21 @@ function createFeedItem(feedItem, trx) {
 
   trx = trx || knex;
 
-  return trx.returning('id').insert(dbRow).into('feed_items')
+  return trx.returning('*').insert(dbRow).into('feed_items')
```

Comment:
No need to fetch all columns as none of them are used. Minor optimisation but reduces the amount of data returned from the database